### ISSUE TITLE
Support `git clone --recursive`

### ISSUE THREAD

**Issue by oalders** [genehack/app-gitgot#68]
@genehack thanks for all of your work on this module.  Unless I've misunderstood, it doesn't look like I can `got clone --recursive $url`.  Have I missed something?

**Comment by genehack** [comment]
Nope, that's not supported because nobody has yet mentioned needing to scratch that itch (the few external repos I have that use submodules I just do by hand). Happy to entertain patches or just leave this open as a feature request; I'm not sure when I might get around to this myself.

**Comment by oalders** [comment]
Thanks and understood.  I don't think I currently have the bandwidth for a pull request, but I at least wanted to establish whether or not this is currently possible.  Maybe submodules aren't so popular.  :)

**Comment by genehack** [comment]
There's no reason it couldn't get added as a flag to the `clone` subcommand -- just needs somebody to rub the right amount of tuits on the code. 8^) I'll leave this issue open on the off chance a PRC participant wanders by or something...

**Comment by yanick** [comment]
I recommend turning off the lights, facing a mirror and whispering 'yenzie' 3 times... >.>
